src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { Meta } from "@storybook/react/types-6-0";
+import Dropdown from "./Dropdown";
+import { withDesign } from "storybook-addon-designs";
+import { ExampleCard } from "../Card/Card.stories";
+
+export default {
+  title: "Dropdown",
+  component: Dropdown,
+  decorators: [withDesign],
+} as Meta;
+
+// Set up the reusable template to create multiple stories for the
+// Dropdown component.
+const DropdownTemplate = ({ ...args }) => (
+  <>
+    <Dropdown {...args} inputId={`${args.inputId}`} />
+    <ExampleCard />
```

Comment:
I think this can be removed, along with its import on L5.

Reply:
@hokei suggested having some content below to show that it opens on top of it, rather than pushing content down, like an accordion. Thoughts?

Reply:
Oh, that makes perfect sense, I think I'm just not seeing it in the story for some reason... are you seeing it locally?

Reply:
Nvrmnd—seeing it now!